plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -79,6 +95,19 @@ const (
 // For example:
 // https://github.com/kubernetes/kubernetes/blob/36a218e/plugin/pkg/scheduler/factory/factory.go#L422
 
+// IMPORTANT: this list contains the ordering of the predicates, if you develop a new predicates
+// it is mandatory to add its name on this list.
+// otherwise it won't be processed, see generic_scheduler#podFitsOnNode()
+var (
+	predicatesOrdering = []string{CheckNodeConditionPred,
```

Comment:
Need to explain how the order is defined in the comment.

Reply:
I'll explain briefly how the order is defined and point to design doc

Reply:
Yes, please. This is very important because we need to make sure anyone want to add new predicate will not break your design here.

Reply:
And as Bobby suggest, a test verify is needed to ensure it.